osi_featuredata.proto

```diff
@@ -0,0 +1,260 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_common.proto";
+
+package osi;
+
+//
+// \brief Interface for sensor data, in contrast to interpreted data after object hypothesis and tracking.
+//
+// All information regarding the environment is given with respect to the sensor coordinate system specified in \c DetectionHeader::mounting_position. 
+// When simulating multiple sensors, each sensor has an individual copy of FeatureData in its own reference frame. This allows an independent treatment of the sensors.
+//
+message FeatureData
+{
+     // Lidar detection list for multiple lidar sensors.
+     //
+     repeated DetectionLidarList detection_lidar_list = 1;
+
+     // Radar detection list for multiple lidar sensors.
+     //
+     repeated DetectionRadarList detection_radar_list = 2;
+}
+
+//
+// \brief The header attributes of each detection.
+//
+message DetectionHeader
+{
+   // Version number of detecion list messages.
```

Comment:
I think this needs more documentation, since it does not become clear the version of what is meant here (in the OSI context, we already have the OSI version number completely describing the format, so without further comments the intended meaning does not get clear enough I think).